providers/exasol/pyproject.toml

```
@@ -65,6 +65,13 @@ dependencies = [
     'pandas>=2.2.3; python_version >="3.13"',
 ]
 
+# The optional dependencies should be modified in place in the generated file
+# Any change in the dependencies is preserved when the file is regenerated
+[project.optional-dependencies]
+sqlalchemy = [
+    "sqlalchemy>=1.4.0",
```

```suggestion
    "sqlalchemy>=1.4.49",
```

I think we should add same specification version as in airflow. This will make it easier to keep it in sync - we do need asyncio here, so I guess ^^ will be correct.

---

My bad.  Updated.  Thank you for pointing that out.